experiments/gail_benchmark_config.csv

```diff
@@ -1,36 +1,36 @@
-env_config,rollout_glob,n_gen_steps_per_epoch,n_epochs,n_demonstrations
-cartpole,CartPole-v1.pkl,5000,300,1
-cartpole,CartPole-v1.pkl,5000,300,4
-cartpole,CartPole-v1.pkl,5000,300,7
-cartpole,CartPole-v1.pkl,5000,300,10
-mountain_car,MountainCar-v0.pkl,5000,300,1
-mountain_car,MountainCar-v0.pkl,5000,300,4
-mountain_car,MountainCar-v0.pkl,5000,300,7
-mountain_car,MountainCar-v0.pkl,5000,300,10
-acrobot,Acrobot-v1.pkl,5000,300,1
-acrobot,Acrobot-v1.pkl,5000,300,4
-acrobot,Acrobot-v1.pkl,5000,300,7
-acrobot,Acrobot-v1.pkl,5000,300,10
-half_cheetah,HalfCheetah-v2.pkl,50000,500,4
-half_cheetah,HalfCheetah-v2.pkl,50000,500,11
-half_cheetah,HalfCheetah-v2.pkl,50000,500,18
-half_cheetah,HalfCheetah-v2.pkl,50000,500,25
-hopper,Hopper-v2.pkl,50000,500,4
-hopper,Hopper-v2.pkl,50000,500,11
-hopper,Hopper-v2.pkl,50000,500,18
-hopper,Hopper-v2.pkl,50000,500,25
-walker,Walker2d-v2.pkl,50000,500,4
-walker,Walker2d-v2.pkl,50000,500,11
-walker,Walker2d-v2.pkl,50000,500,18
-walker,Walker2d-v2.pkl,50000,500,25
-ant,Ant-v2.pkl,50000,500,4
-ant,Ant-v2.pkl,50000,500,11
-ant,Ant-v2.pkl,50000,500,18
-ant,Ant-v2.pkl,50000,500,25
-humanoid,Humanoid-v2.pkl,50000,1500,80
-humanoid,Humanoid-v2.pkl,50000,1500,160
-humanoid,Humanoid-v2.pkl,50000,1500,240
-reacher,Reacher-v2.pkl,50000,200,4
-reacher,Reacher-v2.pkl,50000,200,11
-reacher,Reacher-v2.pkl,50000,200,18
-reacher,Reacher-v2.pkl,50000,200,25
+env_config,env_name,n_gen_steps_per_epoch,n_epochs,n_demonstrations
```

Comment:
It's unfortunate that I have to keep track of the `env_name` right now to access the logs. It would be nice if on the python side of things, I could save artifacts into `$LOG_DIR/{env_config}` instead of `$LOG_DIR/{env_name}`.

Migrating to ray and taking advantage of `run.results` wouldn't automatically solve this issue because Phase 2 (this script) usually won't run straight after Phase 1. We will typically download a zip file to skip Phase 1.

Reply:
Why not just set `log_dir="${OUTPUT_DIR}/{env_config}` in the `experiments/gail_benchmark.sh`? All `log_dir` in the Sacred config is log root plus environment plus timestap. You already have a timestamp in OUTPUT_DIR so we don't need that again.

I'd also be fine with replacing `env_name` in `log_dir` with `exp_name`, which should default to `env_name` but could be overidden in particulra named configs. We just want a sensible default for `log_dir`.

Reply:
Why not just set `log_dir="${OUTPUT_DIR}"/{env_config}` from inside the bash script? The `log_dir` in Sacred is just meant to be a sensible default (log_root + env_name + timestamp), and since we already have a timestamp in OUTPUT_DIR seems fine to override.

You could also change this on the Python end, by e.g. changing `env_name` in the default `log_dir` to `exp_name`, and then setting `exp_name = env_name` by default but letting named configs override it. Seems unnecessary for now though.